cumulus_library/cli.py

```diff
@@ -113,73 +113,126 @@ def export_all(self, study_dict: Dict):
+        for child_path in parent_path.iterdir():
+            if child_path.is_dir():
+                manifest_studies[child_path.name] = child_path
+            elif child_path.name == "manifest.toml":
+                manifest_studies[parent_path.name] = parent_path
     return manifest_studies
 
 
 def run_cli(args: Dict):  # pylint: disable=too-many-branches
     """Controls which library tasks are run based on CLI arguments"""
-    builder = StudyBuilder(
-        args["s3_bucket"],
-        args["region"],
-        args["workgroup"],
-        args["profile"],
-        args["database"],
-    )
-    if args["verbose"]:
-        builder.verbose = True
+    if not args["build"] and not args["export"] and not args["create"]:
+        sys.exit(
+            (
+                "Expecting at least one of build, export or create as arguments. "
+                "See `cumulus_library --help` for more information"
+            )
+        )
+
+    if args["create"]:
+        create_template(args["path"])
 
-    # here we invoke the cursor once to confirm valid connections
-    builder.cursor.execute("show tables")
+    if args["build"] or args["export"]:
+        builder = StudyBuilder(
+            args["s3_bucket"],
+            args["region"],
+            args["workgroup"],
+            args["profile"],
+            args["database"],
+        )
+        if args["verbose"]:
+            builder.verbose = True
 
-    if not args["build"] and not args["export"]:
-        sys.exit("Neither build nor export specified - exiting with no action")
-    study_dict = get_study_dict()
+        study_dict = get_study_dict(args["study_dir"])
+        # here we invoke the cursor once to confirm valid connections
+        builder.cursor.execute("show tables")
 
-    if args["build"]:
-        if "all" in args["target"]:
-            builder.clean_and_build_all(study_dict)
-        else:
-            for target in args["target"]:
-                if target in study_dict:
-                    builder.clean_and_build_study(study_dict[target])
+        if args["build"]:
+            if "all" in args["target"]:
+                builder.clean_and_build_all(study_dict)
+            else:
+                for target in args["target"]:
+                    if target in study_dict:
+                        builder.clean_and_build_study(study_dict[target])
 
-    if args["export"]:
-        if "all" in args["target"]:
-            builder.export_all(study_dict)
-        else:
-            for target in args["target"]:
-                builder.export_study(study_dict[target])
+        if args["export"]:
+            if "all" in args["target"]:
+                builder.export_all(study_dict)
+            else:
+                for target in args["target"]:
+                    builder.export_study(study_dict[target])
 
-    # returning the builder for ease of unit testing
-    return builder
+        # returning the builder for ease of unit testing
+        return builder
```

Comment:
You might not want to indent the return here

Reply:
I explicitly do so i don't have to worry about checking if i'm in the build/export group of functions. I don't need this for testing create, and don't want to worry about forcing some create logic to output something that adds nicely to the builder.

Reply:
Yeah I had missed that you only create the builder in this branch of the code.